app/controllers/ipaddresses_controller.rb

```diff
@@ -0,0 +1,60 @@
+class IpaddressesController < ApplicationController
+  include IpUtilities
+
+  def index
```

Comment:
INFO: Metrics/PerceivedComplexity: Perceived complexity for index is too high. [11/8]